client/lib/titan/test/lib/fixtures.ts

```diff
@@ -0,0 +1,114 @@
+import type { ResponseDomain, EmailCost } from 'calypso/lib/domains/types';
+
+const exampleDomain: ResponseDomain = {
```

Comment:
This is also just a drive-by observation, but could we maybe use `DOMAIN_PRIMARY` from `client/state/sites/domains/test/fixture/index.js` as the base object here instead?

I haven't inspected any potential differences between the two, but it's another dummy instance of `ResponseDomain`, also used for testing.

Reply:
Wow, I haven't found it previously, thank you ❤️